tests/x11/yast2_snapper.pm

```diff
@@ -35,6 +35,12 @@ use y2_module_consoletest;
 sub run {
     my $self = shift;
 
+    # Make sure that the module runs on graphical mode
+    unless (check_screen 'generic-desktop', 0) {
+        select_console 'x11';
+        $self->handle_displaymanager_login() if (check_screen 'linux-login', 0);
+    }
```

Comment:
Do you still need this condition now?

Reply:
I don't need it in the particular scenario, but since I spend time on this and it's tested and works, why remove it?

Reply:
it makes the module more robust as we were discussing offline, for example in case someone would like to schedule it in another test suite not using desktop. What we should do as a good practice is to leave some quick verification where we try this specific path of code, for example scheduling only module in some invented textmode test suite.

Reply:
done